### PR TITLE
Mouse hover support for inventory screens

### PR DESCRIPTION
**Pull request by katemonster33**
…inventory screen as well as the inventory multi-selector.

#### Summary
Features "Made it so that inventory items highlight when hovered over with the mouse"

#### Purpose of change

When I first installed the game, I was very confused by how the mouse worked as expected on the main menu, but not anywhere else. On the inventory screen, though I found that clicking items selects them as expected, without seeing the items become highlighted as the mouse is moved over them, it appeared that the dialog did not support the mouse at all. I feel this change adds UI consistency. Relevant to #58106, does not fully fix it.

#### Describe the solution

Registered actions for MOUSE_MOVE and COORDINATE on the action handler. COORDINATE turns on mouse hover events, but I found that I needed to register a separate action for MOUSE_MOVE to see the event propagate up to the on_input functions.

#### Describe alternatives you've considered
n/a

#### Testing
I loaded both the pick-selector and the multi-selector inventory views, by using the 'i' key and 'd' keys, respectively. I am not 100% familiar with all the screens in CDDA yet but it didn't look like this was used in any other places.

#### Additional context
n/a

### CODE REVIEW

**Comment by dchieh222** [comment]
This seems like it'll finally lead to tooltips!

**Comment by katemonster33** [comment]
Added a commit... noticed regressions in mouse support in pocket examiner (inventory_examiner) screen and the inventory compare screen, fixing the bugs has caused mouse hover support to now be working as intended in those screens as well.

File: src/inventory_ui.cpp [hunk 1]
```diff
@@ -3525,7 +3531,9 @@ void inventory_multiselector::on_input( const inventory_input &input )
 
     if( input.entry != nullptr ) { // Single Item from mouse
         highlight( input.entry->any_item() );
-        toggle_entries( count );
+        if( input.action == "SELECT" ) {
+            toggle_entries( count );
+        }
     } else if( input.action == "TOGGLE_NON_FAVORITE" ) {
```
Comment (andrei8l) [inline src/inventory_ui.cpp:3537]:
```suggestion
    }
    if( input.action == "TOGGLE_NON_FAVORITE" ) {
```
This should make clicking on chevron (from #64750) work for multiselectors too

**Comment by leemuar** [comment]
well done, thanks! one step closer to better mouse support

**Pull request merged**